test/pdfs/xfa_imm5257e.pdf.link

```diff
@@ -0,0 +1 @@
+https://web.archive.org/web/20210521163640/https://www.canada.ca/content/dam/ircc/migration/ircc/english/pdf/kits/forms/imm5257e.pdf
```

Comment:
Please add a blank line at the end of the file (to avoid the big red marker here in the Github diff).

Reply:
Fixed.